Gems/EMotionFX/Code/Source/Editor/ColliderContainerWidget.cpp

```diff
@@ -326,8 +326,26 @@ namespace EMotionFX
         {
             return;
         }
+        if (!m_editor->HasDisplayedNodes())
+        {
+            hide();
+        }
+        else
+        {
+            show();
+        }
```

Comment:
I think this can be just
```suggestion
        setVisible(m_editor->HasDisplayedNodes());
```